.github/workflows/update-readme.yml

```
@@ -0,0 +1,33 @@
+name: Data Fetch
+
+on:
+    schedule:
+        - cron: "0 8 * * *" # Every day at 1am PDT
```

If we add `workflow_dispatch` then we can [run this manually](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/):

```suggestion
on:
    schedule:
        - cron: "0 8 * * *" # Every day at 1am PDT
    workflow_dispatch:
```

---

Sounds good to me. @harish-sethuraman what do you think about committing this change?

---

Done. Seems like this should work. I don't have write access to this repo so I would not be able to trigger workflow manually?

---

I think it can be triggered on the default (`main`) branch only. We can try it out when this PR is merged.

---

Yes that's correct, it can only be triggered on the `main` branch once its merged.